packages/react/src/components/CardLink/__stories__/Card.stories.js

```diff
@@ -4,17 +4,20 @@ import { withKnobs, select, text } from '@storybook/addon-knobs';
 import { CardLink } from '../';
 import { ArrowRight20 } from '@carbon/icons-react';
 import readme from '../README.md';
+import { DDS_CARD_LINK } from '../../../internal/FeatureFlags';
 
 import './index.scss';
 
-storiesOf('Card', module)
-  .addDecorator(withKnobs)
-  .addDecorator(Story => {
-    return (
-      <div className="bx--grid">
-        <div className="bx--row">
-          <div className="bx--col-sm-2 bx--col-md-3 bx--col-lg-6 bx--col-xlg-4 bx--no-gutter">
-            <Story />
+if (DDS_CARD_LINK) {
```

Comment:
@mkothur we have removed the feature flag for CardLink.

Reply:
Okay, I will revert back to that.